components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/AdminForcedPasswordResetConfigImpl.java

```diff
@@ -135,4 +147,27 @@ public Properties getDefaultPropertyValues(String tenantDomain) throws IdentityG
         return null;
     }
 
+    private Property getPropertyObject(String type, String regex) {
+
+        Property property = new Property();
+        property.setType(type);
+        if (regex != null) {
+            property.setRegex(regex);
+        }
+        return property;
+    }
+
+    @Override
+    public Map<String, Property> getMetaData() {
+
+        Map<String, Property> meta = new HashMap<>();
+        meta.put(IdentityRecoveryConstants.ConnectorConfig.ENABLE_ADMIN_PASSWORD_RESET_WITH_RECOVERY_LINK,
```

Comment:
One of these should be selected at once. Can get that info from these details?

Reply:
In this fix it doesn't

Reply:
Fixed Added 

Fixed in commit - 157378150349a3707db8435f8ed64041515e6ad1

Other relevant PR(s) -  https://github.com/wso2/carbon-identity-framework/pull/3565